packages/client/.env.template

```diff
@@ -22,3 +22,8 @@ REACT_APP_DISABLE_CACHE=true
 # MongoDB config
 # When using MongoDB as cache, large results will be paginated and split into multiple requests. Use this to set a limit for results per page/request.
 PAGE_LIMIT=30
+
+#variables for enabling and disabling region recommendation and forecast feature
+REACT_APP_ENABLE_REGION_RECOMMENDATION_FEATURE = true
+REACT_APP_ENABLE_FORECAST_FEATURE = true
+REACT_APP_CARBON_AWARE_SDK_BASE_URL = http://localhost:5073
```

Comment:
I believe these variables make sense, but should probably live in the API/backend along with the SDK calls. 

For `REACT_APP_ENABLE_FORECAST_FEATURE`, this should be renamed or made more specific so that it's not confused with the Forecast component on the recommendations page. 

In regards to, `REACT_APP_CARBON_AWARE_SDK_BASE_URL` does this require a locally or self-hosted version of the SDK? If so, we should probably have documentation on how to set this up for CCF